src/EventTimer.ts

```diff
@@ -141,7 +145,7 @@ export class EventTimer {
 					timingLabel: 'Commercial start parse time',
 				},
 				{
-					timingVariable: 'commercialEnd',
+					timingVariable: 'commercialModulesLoaded',
```

Comment:
What is the impact of changing the name of the `timingVar` we pass to `window.ga`?

Reply:
There's no harm in renaming the label we send to Google Analytics.